apps/iotune/iotune.cc

```diff
@@ -903,14 +923,8 @@ int main(int ac, char** av) {
                 rates = iotune_tests.get_sharded_worst_rates().get();
                 fmt::print("{} IOPS{}\n", uint64_t(read_iops.iops), accuracy_msg());
 
-                struct disk_descriptor desc;
-                desc.mountpoint = mountpoint;
                 desc.read_iops = read_iops.iops;
-                desc.read_bw = read_bw.bytes_per_sec;
-                desc.read_sat_len = read_sat;
                 desc.write_iops = write_iops.iops;
-                desc.write_bw = write_bw.bytes_per_sec;
-                desc.write_sat_len = write_sat;
                 disk_descriptors.push_back(std::move(desc));
```

Comment:
In case run_iops_only is true, the `desc` thing is empty, isn't it? In that case, what will be there in the io-properties.yaml file after this push-back?

Reply:
`desc` is filled with 0s, so the numbers except iops will be 0 in case `run_iops_only` is true. We can maybe warn that bw numbers are not to be used or something like that, but other than that, not sure what we can do better. The `io-properties.yaml` file is not meant to have any meaning when this option is passed. Another option would be to not dump the file at all if the option is true?